ui/inspectors/system-section.reel/updates/updates.reel/updates.js

```diff
@@ -71,7 +71,12 @@ exports.Updates = Component.specialize(/** @lends Updates# */ {
 
     handleCheckDownloadAction: {
         value: function() {
-            this._updateService.checkAndDownload();
+            var self = this;
+            this._updateService.checkAndDownload().then(function() {
+                return self._updateService.getInfo();
+            }).then(function(info) {
+                self.info = info;
```

Comment:
For future reference: this turned out not to be useful, so instead we went back to the previous behavior of checkfetch also returning the new update data.